pom.xml

```diff
@@ -30,6 +30,11 @@
             <artifactId>kotlin-unsigned</artifactId>
             <version>v3.1.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.jitsi</groupId>
+            <artifactId>jitsi-utils</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
do we not have versions of jitsi-utils yet?

Reply:
We do, but we don't have the version that this PR needs yet. I'll update pom.xml once we merge the PR in jitsi-utils.